docs/_methods.ts

```diff
@@ -3,14 +3,20 @@ import { Props as MethodProps } from "./components/method";
 export const methods: { [name: string]: MethodProps } = {
   ref: {
     name: "ref()",
-    signatures: ["ref(tableName)"],
+    signatures: ["ref(name)", 'ref({schema: "schema", name: "name"})'],
```

Comment:
don't worry about changing this file, it's super out of date and not used in the documentation anyway. we should probably delete it, but i think we're still thinking about the right way of doing this documentation.

Reply:
Done